channels/ON-CHAIN.md

```diff
@@ -173,19 +184,36 @@ In order to close a channel unilaterally, a participant has to send a
 responding but can also be used by an malicious peer trying to close a channel
 with a state that hasn't been agreed on by all participants.
 
-With the inclusion of this transaction on chain, the timer, during which
+With the inclusion of this transaction on-chain, the timer, during which
 disputes in the form of `channel_slash` will be considered, is started.
 
 
 Serialization defined [here](../serializations.md#channel-close-solo-transaction)
 
 
-- `channel_id`:
-- `from`
+- `channel_id`: channel id as recorded on-chain
+- `from`: participant of the channel that posts the closing transaction
+- `payload`: empty or a transaction proving that the proof of inclusion is part of the channel
+- `poi`: proof of inclusion
 - `ttl`
-- `nonce`
+- `nonce`: taken from the `from`'s account
 - `fee`
 
+Proof of inclusion represents a subset of the internal channel state. At the
+bare minimum it has to include all accounts and their balances but can also
+include contracts and contract calls.
+
+Payload is a valid transaction that has:
```

Comment:
Listing which transaction types are allowed here would help reader.

Reply:
Later I read:

> If the transaction type is channel_create/channel_deposit/channel_withdraw/channel_offchain it MUST be co-signed.

that leaves the possibility that other transaction types may be here.